docs/conf.py

```diff
@@ -2,17 +2,21 @@
 import importlib.metadata
 import datetime
 import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../src/"))
```

Comment:
Another way is to specify `.` (project root) as a dependency.

Reply:
You took another path with setting vars. I don't have an opinion.